xml/System.Net.Http/HttpRequestOptions.xml

```diff
@@ -684,11 +693,11 @@ This member is an explicit interface member implementation. It can be used only
         </Parameter>
       </Parameters>
       <Docs>
-        <typeparam name="TValue">To be added.</typeparam>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TValue">The type of the HTTP value as defined by <code>key</code> parameter.</typeparam>
+        <param name="key">Strongly typed key to get the value of HTTP request option. For example <code>new HttpRequestOptionsKey&lt;bool&gt;("WebAssemblyEnableStreamingResponse")</code></param>
+        <param name="value">Returns the value of HTTP request option.</param>
+        <summary>Gets the value of a given HTTP request option.</summary>
+        <returns>True, if an option is retrieved.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the collection contains an element with the specified key; otherwise, <see langword="false" />.</returns>
```